exercise4/playbook.yml

```
@@ -0,0 +1,71 @@
+- name: run webservers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+
+    - name: trello_api_key
+      prompt: What is your trello api key?
+      private: false
+
+    - name: trello_api_token
+      prompt: What is your trello api token?
+      private: false
+
+  tasks:
+    - name: Install the latest version of Git
+      become: yes
+      ansible.builtin.yum:
+        name: git
+        state: latest
+
+    - name: Install Python v3.11
+      become: yes
+      ansible.builtin.yum:
+        name: python311
+        state: latest
```

Not important, but worth noting that we can combine these if we want:
```suggestion
    - name: Install the latest version of Git & Python v3.11
      become: yes
      ansible.builtin.yum:
        name:
          - git
          - python311
        state: latest
```